collectors/builtin/services_startup.py

```diff
@@ -38,17 +37,26 @@ def print_metric(self, ts, value, tags=""):
 
     def __init__(self, config, logger, readq):
         super(ServicesStartup, self).__init__(config, logger, readq)
-        self.services = self.get_config('services', 'alertd, datanode').split(',')
+        self.initialize = True
+        self.services_before = []
```

Comment:
Rename services_before to last_started_services, and services_after to curr_started_services.